master/buildbot/test/util/migration.py

```diff
@@ -82,5 +84,17 @@ def upgrade_thd(engine):
                 schema.runchange(version, change, 1)
         d.addCallback(lambda _: self.db.pool.do_with_engine(upgrade_thd))
 
+        def check_table_charsets(engine):
```

Comment:
I think it should be `check_table_charsets_thd` or something like this. It's being run in thread, right?